consensus/src/processes/pruning_proof/build.rs

```diff
@@ -209,6 +207,33 @@ impl PruningProofManager {
         )
     }
 
+    /// Given two hashes (both are assumed to exist in the passed ghostdag store),
+    /// find their latest common ancestor. The latest common ancestor is minimum block
+    /// such that both 'a' and 'b' are either equal to it or in its future.
+    /// This ancestor must also be in the selected parent chain of 'a'.
+    ///
+    /// Additional Notes:
+    /// - 'a' and 'b' are expected to be hashes acquired via block_at_depth.
+    ///   By virtue of this, they are both chain block hashes.
+    fn find_latest_common_ancestor_in_store(
```

Comment:
Maybe remove the `latest` part? This function is mostly a heuristic to find a late common ancestor, not the latest necessarily.